src/osdag/design_type/compression_member/Column.py

```diff
@@ -952,26 +963,26 @@ def design_column(self):
+                                                                                                    self.section_property.flange_thickness,
+                                                                                                    cross_section='Rolled I-sections',
+                                                                                                    section_type='Hot rolled')['y-y']
                     else:
                         self.buckling_class_zz = IS800_2007.cl_7_1_2_2_buckling_class_of_crosssections(self.section_property.flange_width,
-                                                                                                       self.section_property.depth,
-                                                                                                       self.section_property.flange_thickness,
-                                                                                                       cross_section='Welded I-section',
-                                                                                                       section_type='Hot rolled')['z-z']
+                                                                                                    self.section_property.depth,
+                                                                                                    self.section_property.flange_thickness,
+                                                                                                    cross_section='Welded I-section',
+                                                                                                    section_type='Hot rolled')['z-z']
                         self.buckling_class_yy = IS800_2007.cl_7_1_2_2_buckling_class_of_crosssections(self.section_property.flange_width,
-                                                                                                       self.section_property.depth,
-                                                                                                       self.section_property.flange_thickness,
-                                                                                                       cross_section='Welded I-section',
-                                                                                                       section_type='Hot rolled')['y-y']
+                                                                                                    self.section_property.depth,
+                                                                                                    self.section_property.flange_thickness,
+                                                                                                    cross_section='Welded I-section',
+                                                                                                    section_type='Hot rolled')['y-y']
```

Comment:
These changes appear to be identical to the original. Why has git caught these?

Reply:
Because there is an indentation that was removed in the new code snippet I suppose.